### DISCUSSION TITLE
How to use Quarkus to add Extensions for Keycloak

### DISCUSSION THREAD

**Discussion by mrp4sten** [keycloak/keycloak#33664] [Archive]
Keycloak now use the Quarkus for improve the extensibility and now I want to use the extension Google Cloud Services - Logging for the keycloak version 20.0.2

I try to add the extension when I'm on ~/keycloak or ~/keycloak/quarkus path 

```shell
../mvnw quarkus:add-extension -Dextensions="quarkus-google-cloud-logging"

# or

mvn quarkus:add-extension -Dextensions="quarkus-google-cloud-logging"
```

But I every time have this error:

```log
[ERROR] No plugin found for prefix 'quarkus' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/home/mavor/.m2/repository), jboss-public-repository (https://repository.jboss.org/nexus/content/groups/public/), central (https://repo.maven.apache.org/maven2)]
```

Im using Java 11 btw, and also I've already have installed quarkus on cli using sdkman

**Comment by codespearhead** [comment]
You should upgrade to the latest stable version as soon as it becomes available, as only the latest release will receive updates [1][2].

Could you please close this discussion [3] in favor of [4][5]?

[1] https://github.com/keycloak/keycloak/discussions/29126
[2] https://github.com/keycloak/keycloak/discussions/29126#discussioncomment-9257666
[3] https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion
[4] https://github.com/quarkusio/quarkus/discussions/41056#discussioncomment-9882006
[5] https://github.com/quarkusio/quarkus/discussions/41056

**Comment by mabartos** [comment]
Hey guys, I've created a tool that provides the capability to add arbitrary Quarkus/Quarkiverse extensions to your Keycloak deployments.

You can check it here: https://github.com/mabartos/keycloak-quarkus-extensions
